content/docs/command-reference/init.md

```diff
@@ -32,67 +32,59 @@ that are hidden from the user. This directory is automatically staged with
 
-`--subdir` is mostly used in the scenario of a
-[monorepo](https://en.wikipedia.org/wiki/Monorepo), but also can be used in
-other workflows when such isolation and/or advanced granularity is needed.
+This option is mostly useful in the scenario of a
+[monorepo](https://en.wikipedia.org/wiki/Monorepo) (Git repository split into
+several project directories), but can also be used with other patterns when such
+isolation is needed. `dvc init --subdir` mitigates the issues of initializing
+DVC in the Git repository root:
 
-Let's imagine we have an existing Git repository that is split into sub-projects
-(monorepo). In this case `dvc init --subdir` can be run in one or many
-sub-projects to mitigate the issues of initializing in the Git repository root:
+- Repository maintainers might not allow a top level `.dvc/` directory,
+  especially if DVC is being used by several sub-projects (monorepo).
 
-- Repository maintainers might not allow extra `.dvc` top level directory,
-  especially if DVC is being used by a small number of sub-projects.
+- DVC config file, cache directory,
+  [etc.](/doc/user-guide/dvc-files-and-directories) are shared across different
+  sub-projects. This makes it difficult to use different DVC settings,
+  [remote storage](/doc/command-reference/remote) locations, etc.
 
-- Not enough isolation/granularity - DVC config, cache, and other files are
-  shared across different sub-projects. Means that it's not easy to use
-  different remote storages, for example, for different sub-projects, etc.
-
-- Not enough isolation/granularity - commands like `dvc pull`, `dvc checkout`,
-  and others analyze the whole repository to look for `dvc.yaml` or `.dvc` files
-  to download files and directories, to reproduce <abbr>pipelines</abbr>, etc.
-  It can be expensive in the large repositories with a lot of projects.
-
-- Not enough isolation/granularity - commands like `dvc metrics diff`, `dvc dag`
-  and others by default dump all the metrics, all the pipelines, etc.
+- Many DVC commands can explore the whole <abbr>DVC repository</abbr> to find
+  DVC-tracked data and pipelines to work with. This can be undesirable and
+  inefficient for large monorepos.
 
 #### How does it affect DVC commands?
 
-No matter what mode is used, DVC looks for the `.dvc` directory when it starts
-(from the current working directory and up). Location of the found `.dvc`
-directory determines the root of the DVC project. (In case of `--subdir` it
-might happen that Git repository root is located at different path than the DVC
-project root.)
+The <abbr>project</abbr> root defines the possible scope of action for most DVC
+commands (e.g. `dvc repro`, `dvc pull`, `dvc metrics diff`), meaning that only
+`dvc.yaml`, `dvc.lock`, and `.dvc` files under that location are usable by the
+commands. To determine the root of the project, DVC looks for `.dvc/` from the
+current working directory, up. With `--subdir`, the project root will be found
```

Comment:
not sure what Git root has to do with root search

Reply:
This paragraph is about what the project root is, how it's found, how it affects DVC commands, and finally, how --subdir plays into all this:

> With `--subdir`, the project root will be found before the Git repo root, reducing this scope.

Maybe the p is too complicated, I'll split it in 2...

Reply:
Improved explanation in 40512a11.